include/pstl_offload/internal/usm_memory_replacement.h

```diff
@@ -186,17 +207,12 @@ __internal_operator_new(std::size_t __size, std::size_t __alignment, const std::
 
 } // namespace __pstl_offload
 
-#if __linux__
-
-// valloc, pvalloc, __libc_valloc and __libc_pvalloc are not supported
-// due to unsupported alignment on memory page
-
 extern "C"
 {
 
 inline void* __attribute__((always_inline)) malloc(std::size_t __size)
 {
-    return ::__pstl_offload::__errno_handling_internal_aligned_alloc(__size, alignof(std::max_align_t));
+    return ::__pstl_offload::__errno_handling_internal_aligned_alloc(__size, 0);
```

Comment:
May be it will also make sense to save 0 into some constexpr variable with meaningful name and comment (something like `non_extended_alignment_flag`) and use the variable instead of 0 as magic constant. What do you think?

Reply:
Done.